### PR TITLE
Update config.json

### PR DESCRIPTION
**Pull request by ersanyakit**
Original Site : https://roco.finance
Scam Site : hxxp://rocofinance dot com

### CODE REVIEW

File: src/config.json [hunk 1]
```diff
@@ -993,9 +993,11 @@
     "metadusk.io",
     "metabase.finance",
     "metamarc.io",
-    "metapass.fyi"
+    "metapass.fyi",
+    "roco.finance",
```
Comment (devenblake) [inline src/config.json:997]: You have a trailing comma on the end of that last line in the whitelist object; this is a syntax error in the JSON and why the tests aren't passing.

**Review by trn1ty — CHANGES_REQUESTED** [review]
You have a trailing comma on the end of that last line in the whitelist object; this is a syntax error in the JSON and why the tests aren't passing. If you fix it you should be good, it does look like `rocofinance.com` is phishing.

**Review by ersanyakit — COMMENTED** [review]
Fixed.

**Comment by ersanyakit** [comment]
> You have a trailing comma on the end of that last line in the whitelist object; this is a syntax error in the JSON and why the tests aren't passing. If you fix it you should be good, it does look like `rocofinance.com` is phishing.

yea its scam.

**Review by trn1ty — DISMISSED** [review]
Looks good.

**Review by trn1ty — APPROVED** [review]
Re-reviewing after fixing conflicts.

**Pull request merged**